### PR TITLE
Add various missing PKI related changelog entries

### PR DESCRIPTION
**Pull request by stevendpclark**
Add various changelog entries to call out changes that occurred as part of the PKI rotation project to existing behaviors.

### CODE REVIEW

File: changelog/15277.txt [hunk 1]
```diff
@@ -1,11 +1,6 @@
...
-**Allows Multiple Issuer Certificates to enable Non-Disruptive
-Intermediate/Root Certificate Rotation**: This introduces /keys and /issuers
-endpoints to allow import, generation and configuration of any number of keys
-or issuers that can be used to issue and revoke certificates.  Keys and Issuers
-can be referred to by (a) a unique UUID; (b) a name; (c) “default”.  If an
-issuer existed prior to this feature, that issuer will be tagged by a migration
-as “default” to allow backwards compatible calls which don’t specify an issuer.
-Creation of new roles will assume an issuer of “default” unless otherwise
-specified.  This default can be configured at /config/issuers and /config/keys.
+**Non-Disruptive Intermediate/Root Certificate Rotation**: This allows
```
Comment (stevendpclark) [inline changelog/15277.txt:2]: @sgmiller Reworked the feature changelog entry based on a previous comment of yours in the original [big PR](https://github.com/hashicorp/vault/pull/15277/files#r866301293). Let me know if this is what you had in mind.

**Review by kitography — APPROVED** [review]
These look really good and cover everything i wrote down needing to be written.

File: changelog/15277.txt [hunk 2]
```diff
@@ -1,11 +1,6 @@
...
+import, generation and configuration of any number of keys and/or issuers
+within a PKI mount, providing operators to ability to rotate certificates
```
Comment (cipherboy) [inline changelog/15277.txt:4]:
```suggestion
within a PKI mount, providing operators the ability to rotate certificates
```

Comment (stevendpclark) [reply changelog/15277.txt]: Thanks fixed in 15277.txt and within the changelog itself.

**Comment by cipherboy** [comment]
Merged per confirmation from @sgmiller that the updated CL looks good \o/

**Pull request merged**